### PR TITLE
Error content correction

### PR DESCRIPTION
**Pull request by jimcat8**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/release_notes/8_8.md [hunk 1]
```diff
@@ -56,11 +56,11 @@ Some highlights and new features from this release are detailed next.
 Security-related highlights in the latest Rocky Linux 8.8 release are listed below. For a complete list of security related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-security).
 
 * FIPS mode settings in the kernel have been adjusted to conform to the Federal Information Processing Standard (FIPS) 140-3. This change introduces stricter settings to many cryptographic algorithms, functions, and cipher suites.
-* Libreswan IPsec implementation was rebased to version 4.9
-* You can now filter the RPM database with `fapolicyd`.
-* OpenSCAP was rebased to version 1.3.7.
-* Rsyslog TLS-encrypted logging now supports multiple CA files.
-* `systemd-socket-proxyd` service now runs in its own SELinux domain due to an update to the SELinux policy.
+* `libreswan` packages has been rebase to version 4.9.
```
Comment (SergeCroise) [inline docs/release_notes/8_8.md:59]:
```suggestion
* `libreswan` packages have been rebased to version 4.9.
```

File: docs/release_notes/8_8.md [hunk 2]
```diff
@@ -56,11 +56,11 @@ Some highlights and new features from this release are detailed next.
...
+* You can now filter the RPM database with `fapolicyd` packages.
+* `openscap` packages was rebased to version 1.3.7.
```
Comment (SergeCroise) [inline docs/release_notes/8_8.md:61]:
```suggestion
* `openscap` packages were rebased to version 1.3.7.
```

**Comment by SergeCroise** [comment]
Hello @jimcat8 ,
good idea,
using dnf list and dnf info,
we should investigate if the use of plural is more accurate here in the release notes.

**Comment by jimcat8** [comment]
> Hello @jimcat8 , good idea, using dnf list and dnf info, we should investigate if the use of plural is more accurate here in the release notes. e.g. **dnf info libreswan dnf info openscap dnl list | libreswan**

There are regular package as well as devel package, so it is more accurate to use the plural

**Review by sspencerwire — APPROVED** [review]

**Comment by sspencerwire** [comment]
Thanks! This looks good.

**Review by SergeCroise — APPROVED** [review]

**Pull request merged**